lib/ts/controllers/s-popover.ts

```diff
@@ -1,68 +1,86 @@
+
+
 (function () {
     "use strict";
-    Stacks.addController("s-popover", {
-        targets: [],
-        
+    Stacks.application.register("s-popover", class extends Stacks.StacksController {
+        static targets: [];
```

Comment:
```suggestion
        static targets = [];
```

Reply:
Or nothing, since this isn't doing anything.

Reply:
Good eye! Suggestion committed.